readme.md

```diff
@@ -276,18 +276,18 @@ The `targets` array is to contain a list of entities (to be scraped from) keyed
 - `dissect`: Split compound attributes into smaller attributes via REGEX. (optional)
 - `preprocess`: Array of attributes which need to be preprocessed. `[attributeName => callable]` (optional)
 - `remap`: Array of attributes which need to be renamed in order to be saved as target objects. `[attributeName => newName]` (optional)
-- `bad_words`: Any scraps found containing these words will be discarded. (optional)
+- `bad_words`: Any scrapes found containing these words will be discarded. (optional)
```

Comment:
```suggestion
- `bad_words`: Any scrap found containing these words will be discarded. (optional)
```

This is referring to a scrap as defined on line [285](https://github.com/reliqarts/laravel-scavenger/pull/6/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R285)

Reply:
See comment above